crowbar_framework/app/models/backup.rb

```diff
@@ -110,7 +110,14 @@ def upgrade
   end
 
   def cleanup
-    @data.rmtree if @data
+    return unless @data
```

Comment:
maybe its worth to do some sanity checking on `@data`.
It should not contain any path that could destroy the system. Maybe it makes sense to have a positive path filter that e.g. only allows paths below `/x/y/z/`